### ISSUE TITLE
REQUEST: Food Delivery Apps

### ISSUE THREAD

**Issue by el1f** [simple-icons/simple-icons#1955]
- [x] **Deliveroo**
[Logo](https://upload.wikimedia.org/wikipedia/fr/f/f7/Deliveroo_logo.svg) (Wikimedia due to lack of official files)
Color `#00ccbc` (taken from the official website)

- [ ] **Just Eat**
Color: `#fa0029` (Ripped from landing page)
Logo (Ripped from the landing page)
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 165 32.4"><path d="M128.5 15.5l3.9-8.6c0.3-0.7 0.3-0.7 0.8-0.7 0.5 0 0.5 0 0.6 0.8l1.2 8.5H128.5zM139.1 3.5c-0.6-3.4-0.6-3.4-5.4-3.4 -4.5 0-4.4 0-5.8 2.8L117.8 24c-1.5 3-1.2 3.3 2.5 3.3 2.9 0 2.9 0 4.1-2.7l1.6-3.6h9.8l0.5 3.6c0.4 2.7 0.5 2.7 3.4 2.7 3.3 0 3.6-0.2 3-3.5L139.1 3.5z" fill="currentColor"></path><path d="M115.7 5.9c2.4 0 2.4 0 2.9-2.9 0.5-2.9 0.5-2.9-1.7-2.9h-12.1c-4.3 0-4.2 0-4.9 4.3l-3.1 18.1c-0.8 4.7-0.7 4.7 3.6 4.7h11.8c2.5 0 2.4 0 2.9-2.8 0.5-2.9 0.5-2.9-1.7-2.9h-10.2l0.8-5.2h8.5c2.3 0 2.3 0 2.7-2.7 0.4-2.8 0.4-2.8-2.7-2.8H105l0.8-4.9H115.7z" fill="currentColor"></path><path d="M85.9 0.2H69c-2.4 0-2.4 0-2.9 2.9 -0.5 2.9-0.4 2.9 1.7 2.9h5.5l-3.1 17.9c-0.6 3.5-0.5 3.5 2.6 3.5 2.9 0 3 0 3.4-2.5l3.2-18.9h5.3c2.4 0 2.4 0 2.9-2.9C88.1 0.2 88.1 0.2 85.9 0.2" fill="currentColor"></path><path d="M39.5 0.2c-3 0-3 0-3.4 2.5l-2.2 13c-0.6 3.8-1.6 6-5.3 6 -3.6 0-4.9-2.2-4.3-5.6l2.1-12.4c0.6-3.5 0.5-3.5-2.6-3.5 -3 0-3.1 0-3.5 2.4l-2.2 13c-1.3 7.4 3.3 12 9.8 12 7.3 0 11-3.3 12.2-11.1l2.2-12.9C42.8 0.2 42.6 0.2 39.5 0.2" fill="currentColor"></path><path d="M14.4 0.2H7c-2.4 0-2.4 0-2.9 2.8C3.6 5.8 3.6 5.9 5.8 5.9h4.4L7.7 20.2c-0.6 3.2-2 5.4-6.3 6.7C0.3 27.2 0 27.6 0 28.1c0 0.6 0.2 1.1 0.7 2.3 0.7 1.7 1.1 2 1.7 2 0.6 0 1.7-0.3 2.1-0.4 6.1-2 8.6-6.7 9.6-12.3l2.8-16.1C17.4 0.2 17.4 0.2 14.4 0.2" fill="currentColor"></path><path d="M53.2 21.5c2.4-0.1 3.7-1.2 3.7-2.3 0-1.4-1.7-1.9-4-2.6 -4.5-1.4-8.6-3-8.6-7.8 0-5.7 4.7-8.8 10.6-8.8 2.5 0 5.3 0.1 7.1 0.3 1.6 0.2 2 0.7 1.5 3.2C63 6 62.4 6.2 61 6.1c-1.7-0.1-3.9-0.3-6.5-0.3 -3 0-3.9 1.1-3.9 2.1 0 1.3 1.2 2 4 2.8 5.2 1.5 8.6 3.6 8.6 8.1 0 5.3-3.9 8.4-10.2 8.6 -3.4 0.1-6.4-0.1-9-0.5 -2-0.3-2.1-0.3-1.6-3.2 0.5-2.6 0.5-2.8 2.4-2.6C48.5 21.5 50.8 21.6 53.2 21.5" fill="currentColor"></path><path d="M163 0.2h-17c-2.4 0-2.4 0-2.9 2.9 -0.5 2.9-0.4 2.9 1.7 2.9h5.5l-3.1 17.9c-0.6 3.5-0.5 3.5 2.6 3.5 2.9 0 3 0 3.4-2.5l3.2-18.9h5.3c2.4 0 2.4 0 2.9-2.9C165.2 0.2 165.2 0.2 163 0.2" fill="currentColor"></path></svg>
```

- [x] **Uber Eats**
[logo](https://commons.wikimedia.org/wiki/File:Uber_Eats_2018_logo.svg) (Wikimedia due to lack of official files and Uber.design being currently down)
Color: either `#000000` or `#5fb709`, what do you guys think?

**Comment by PeterShaggyNoble** [comment]
Official SVGs can be found at:

- https://www.deliveroo.design/
- https://d2vkuayfhnkplp.cloudfront.net/assets/dist/img/logos/je-logo-v3.svg
- https://d3i4yxtzktqr9n.cloudfront.net/uber-sites/04e09deee72d5fce182103961d90edb8.svg

Just Eat previously used the `J` from that logo as their icon but, since they rebranded, I've only ever seen the multi-coloured square used - sometimes with the wordmark, sometimes without, depending on the size it's to be displayed at.

**Comment by PeterShaggyNoble** [comment]
Closed by #2077, #2078 & #2109
